securesystemslib/util.py

```diff
@@ -455,3 +457,51 @@ def digests_are_equal(digest1: str, digest2: str) -> bool:
       are_equal = False
 
   return are_equal
+
+
+def b64enc(data: bytes) -> str:
+  """
+  <Purpose>
+    To encode byte sequence into base64 string
+
+  <Arguments>
+    data:
+      Byte sequence to encode
+
+  <Exceptions>
+    TypeError: If "data" is not byte sequence
+
+  <Side Effects>
+    None.
+
+  <Return>
+    base64 string
+  """
```

Comment:
I appreciate that you adopt the existing code style here (wrt indentation), but I'd say it's okay to deviate from the rest of the file when it comes to docs. Could you please use the same docstring style as above in metadata.py?

Reply:
Ping

Reply:
Our code style guide has some [notes about docstrings](https://github.com/secure-systems-lab/code-style-guidelines/blob/master/python.md#code-documentation).

TLDR: In new functions/methods/classes, we should use the Google docstring-style, even if the new code is added to an existing module that otherwise uses the old docstring style, which you used here.

Context: The advantage of the Google docstring-style is that there is a Sphinx-parser that can convert it for Readthedocs. And that it is generally more compact, e.g. the `<Purpose>` header is omitted because it is implicit and empty sections are also omitted  (often times this is the `<Side Effects>` section).

Please take a quick look at the style guide linked above and let me know if you have more questions.